warble.cpp

```diff
@@ -87,6 +88,13 @@ int main(int argc, char** argv) {
     request.set_username(FLAGS_user);
     payload->PackFrom(request);
     userClient.Event(EVENT::PROFILE, payload);
+  } else if (!FLAGS_stream.empty()) {
+    // eg: ./warble --user alice --stream "#helloworld"
```

Comment:
maybe add a validator to ensure user input hashtag starts with #

Reply:
thanks, fixed